### DISCUSSION TITLE
Renewed my student status but my copilot is gone

### DISCUSSION THREAD

**Discussion by minge0n** [community/community#142563] [GitHub Education]
### Hi everyone,

I have renewed my student status application and it says the perks are pending. However, I tried the application twice (due to payment information issue), and after the first one was rejected, my copilot access is gone. (I still have GitHub Pro)

I think my previous perks should be available, but it's gone. Is there any way to get it back?

**Comment by Aninooo** [comment]
Sometimes, it can take a few days for perks to refresh and become available after renewing your student status. Keep an eye on the status for any updates in the next couple of days.
